### DISCUSSION TITLE
Who wrote somewhere over the rainbow?

### DISCUSSION THREAD

**Discussion by GameExplorer** [ParanoidUser/yolo#350] [Galaxy Trivia] [answered]
- Hawaii

**Comment by ParanoidUser** [comment]
Israel "IZ" (*won't dare to spell his Hawaiian last name on top of my head... Kamakasomething 😊*)

Grab your Uke and play along 

```
C         Em               F       C
Somewhere over the rainbow way up high
```

P.S. Great question! Keep them coming 👏👏

**Comment by thuongtruong1009** [comment, marked as answer]
The song "Somewhere Over the Rainbow" was written by Harold Arlen (music) and Yip Harburg (lyrics) for the 1939 movie "The Wizard of Oz". The song is sung by Judy Garland in her role as Dorothy, and it has since become one of the most iconic and beloved songs in American popular culture. The song's melody and lyrics evoke a sense of longing and hope, and it has been covered by numerous artists in various styles over the years.

**Comment by peterbaccaro** [comment]
The song "Somewhere Over the Rainbow" was written by composer Harold Arlen and lyricist Yip Harburg.  The song (written in 1938) was written specifically written for the movie "The Wizard of Oz" which was released in 1939.
